### PR TITLE
Add last method

### LINKED ISSUE #38: Add "last" method
The `colleciton.last()` method returns the last item in the collection. It does not remove the item from the collection:

**Example**

```js
const collection = Collect([ 1, 2, 3 ])

await collection.last()

// 3

await collection.all()

// [ 1, 2, 3 ]
```

It wouuld be great to let users pass an (async) testing function as a parameter to the `last` method:

```js
await Collect([
  { id: 1, name: 'marcus' },
  { id: 3, name: 'marcus' }
]).last(async ({ id }) => {
  return await User.findById(id)
})

// { id: 3, name: 'marcus', username: 'marcus' }
```

### PR DESCRIPTION
**Pull request by vufly**
I want to resolve #38

### CODE REVIEW

File: src/collection.js [hunk 1]
```diff
@@ -271,6 +271,28 @@ class Collection {
...
+   * @returns {*} the found value
+   */
+  async last (callback) {
+    if (!callback) {
+      return this.items[this.items.length - 1]
+    }
+
+    if (typeof callback === 'function') {
+      const mapped = await this.filter(callback)
+      return mapped[mapped.length - 1]
```
Comment (marcuspoehls) [inline src/collection.js:291]: Can you please add an empty line before the return statement?

**Review by marcuspoehls — CHANGES_REQUESTED** [review]
@vund-vn Hey Vu, thank you for the PR! Your implementation looks sweet, good job! I’m just requesting a minor refinement: adding an empty line befor the return statement.

File: src/collection.js (hunk 1 above)
Comment (vufly) [reply src/collection.js:291]: Do you mean empty line before every return statement?

Comment (marcuspoehls) [reply src/collection.js:291]: An empty line here:
```suggestion

      return mapped[mapped.length - 1]
```

**Comment by marcuspoehls** [comment]
@vund-vn Thank you! Merged :)

**Review by marcuspoehls — APPROVED** [review]

**Pull request merged**